include/wex/GraphicsController.hpp

```diff
@@ -14,7 +14,7 @@ using Circle = sf::CircleShape;
 /// a wex::GraphicsController.
 struct WindowConfig {
 	std::string windowTitle = "Wex Game";
-	float fps				= 60.0;
+	float fps				= 60.0f;
```

Comment:
What if user wants unlimited framerate? What about vsync? Anti aliasing level?

Reply:
This variable isn't even used right now :stuck_out_tongue: 
I should just get rid of it. Vsync and anti aliasing I'll add when they're actually exposed as useful configurable properties.
For now I want to keep it dumb and simple.